modules/47_regipol/none/datainput.gms

```diff
@@ -9,6 +9,7 @@
 
 * initialize regipol target deviation parameter
 pm_emiMktTarget_dev(ttot,ttot2,ext_regi,emiMktExt) = 0;
+pm_emiMktTarget_dev_iter(iteration, ttot,ttot2,ext_regi,emiMktExt) = 0;
```

Comment:
There is no need to initialize this parameter on the `none` realization, as the parameter should never be used in the model in this case because `cm_emiMktTarget` would be `off`. 
If you set a scenario with `cm_emiMktTarget` values, the realization `regiCarbonPrice` is required to be active.

Reply:
yes, but we would likt to always have all parameters available whish we need to the nash conv. visualization